src/qml/components/KeyboardHandler.qml

```diff
@@ -84,7 +84,7 @@ Item {
 
     Connections {
          target: Qt.inputMethod
-         enabled: visible
+         enabled: visible && !Platform.isAndroid
```

Comment:
Please use Platform.isIos because it's enabled for IOS only, not for all except Android

Reply:
@dmitry-zezik plz update

Reply:
done